other/material/import_material_from_jarvis_db_entry.ipynb

```diff
@@ -1,137 +1,749 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #22.    Poscar(hetero).write_file(filename)
</pre><p>We could consider moving this to a separate function later on, perhaps inside jarvis tools? Creating a heterostructure seems like a good repeatable use case. Of course, there are many nuances to the implementation.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/100/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>